src/main/java/me/xmrvizzy/skyblocker/config/SkyblockerConfig.java

```diff
@@ -146,6 +146,7 @@ public static class General {
         public boolean acceptReparty = true;
         public boolean backpackPreviewWithoutShift = false;
         public boolean hideEmptyTooltips = true;
+        public boolean hideStatusEffectOverlay = true;
```

Comment:
Wouldn't it be better if the default value were set to 'false'? This way, we wouldn't disrupt the vanilla aspect, and everyone could decide for themselves whether the status effect is on or off.

Reply:
i agree on that having it as vanilla by default would be better

Reply:
I put it default to true because the information isn't very useful since skyblock has a lot of custom effects and that doesn't serve much use other than clutter the section